agent/php_nrini.c

```diff
@@ -3100,6 +3100,18 @@ STD_PHP_INI_ENTRY_EX("newrelic.vulnerability_management.composer_api.enabled",
                      newrelic_globals,
                      nr_enabled_disabled_dh)
 
+/*
+ * Cloud relationship settings
+ */
+STD_PHP_INI_ENTRY_EX("newrelic.cloud.aws.account_id",
+                     "1",
```

Comment:
```suggestion
                     "",
```

Reply:
fixed in [5c45a0e](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/5c45a0ef346dbb2bf7e1b28a8ed0de4cbabcbb3f)